### DISCUSSION TITLE
I need to access `useBentoConfig()`.

### DISCUSSION THREAD

**Discussion by mperrando** [buildo/bento-design-system#833] [Q&A] [answered]
I made a custom `Table` cell component, where I need to get the current config of the `Table` in use.

If my understanding is right, i could get this using
```ts
const config = useBentoConfig().table;
```

But the function `useBentoConfig` is not exported.

Can you export it and make it accessible by client code?

Otherwise, how can I get the current `Table` options while rendering a custom `Cell`?

Thanks.

**Comment by veej** [comment, marked as answer]
Hi @mperrando,

Exposing the `useBentoConfig` seems a good idea for custom components implemented in app. I'll send a PR soon

**Reply by veej** [reply]
https://github.com/buildo/bento-design-system/pull/834
